pandas/core/config_init.py

```diff
@@ -319,7 +320,7 @@ def is_terminal():
 
 
 with cf.config_prefix("display"):
-    cf.register_option("precision", 6, pc_precision_doc, validator=is_int)
+    cf.register_option("precision", 6, pc_precision_doc, validator=is_pos_int())
```

Comment:
Hmm I don't think this is correct. Isn't this evaluating `is_pos_int()` and returning None, so essentially doing `validator=None`?

Reply:
This was the correct way to call it, but i didn't really like it as it looked different from the calls to is_int and is_text.  Fixed with the latest commit.